eos-tech-support/eos-procmem-sample

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+declare -A rss_sums
+declare -A pss_sums
+
+if [ $EUID != 0 ]; then
+    echo "permission denied: must be run as root" >&2
+    exit 1
+fi
+
+for pid in $(ps -ef | awk '{print $2}'); do
+    if [ -f /proc/$pid/smaps ]; then
+    cmd=$(cat /proc/$pid/comm)
+        rss=$(awk 'BEGIN {i=0} /^Rss/ {i = i + $2} END {print i}' /proc/$pid/smaps)
+        pss=$(awk 'BEGIN {i=0} /^Pss/ {i = i + $2 + 0.5} END {print i}' /proc/$pid/smaps)
```

Comment:
I'm pretty sure that this `+ 0.5` is incorrect.  It appears to me that the Pss values are integers to begin with, and you are summing an extra 0.5 for each Pss entry in the file, effectively reporting values that are quite a bit higher than they should be.

Reply:
You're right. Fixed in the new version.